pyodbc_sa/__init__.py

```diff
@@ -19,11 +19,11 @@
 
 __version__ = '0.3.5'
 
-from . import ibm_db, pyodbc, base, zxjdbc
+from . import pyodbc, base
```

Comment:
Let's please rename pyodbc to something else. Before, I would suffixed them all with backend (eg. pyodbc_backend), but since we now only have one, backend on its own should be fine.